Build error "use of undeclared identifier 'va_start'"

Reported by TimChen at Discord.

```
*** ninja -C build
ninja: Entering directory `build'
[1/10] Compiling C++ object src/libzukou.so.p/logger.cc.o
FAILED: src/libzukou.so.p/logger.cc.o 
clang++ -Isrc/libzukou.so.p -Isrc -I../src -I../include -Iprotocol -I/usr/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -Werror -std=gnu++17 -O2 -g -D_GNU_SOURCE -DWL_HIDE_DEPRECATED -fvisibility=hidden -Wstrict-prototypes -Wno-gnu-zero-variadic-macro-arguments -fPIC -include config.h -MD -MQ src/libzukou.so.p/logger.cc.o -MF src/libzukou.so.p/logger.cc.o.d -o src/libzukou.so.p/logger.cc.o -c ../src/logger.cc
../src/logger.cc:9:3: error: use of undeclared identifier 'va_start'
  va_start(args, format);
  ^
../src/logger.cc:11:3: error: use of undeclared identifier 'va_end'
  va_end(args);
```

--- a/src/logger.h
+++ b/src/logger.h
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <stdarg.h>
 #include <stdio.h>
 
 namespace zukou {

